internal/services/applications/schema.go

```diff
@@ -51,6 +51,7 @@ func schemaOptionalClaims() *schema.Schema {
 								"on_premise_security_identifier",
 								"sam_account_name",
 								"use_guid",
+								"cloud_displayname",
```

Comment:
Can we keep the (alphabetical) ordering here?

Reply:
Sorted alphabetically